### ISSUE TITLE
Video upload to gallery

### ISSUE THREAD

**Issue by cytadela8** [warsztatywww/aplikacjawww#627]
It would be nice if we could upload videos

**Comment by krzys-h** [comment]
Yes, but handling video streaming properly is *hard*. Do you think we can hope the files are encoded such that the browser can seek, buffer etc. without any trouble? This is not always the case, but I hope videos from a phone camera shouldn't be much of an issue and I REALLY don't want to mess with having to transcode them server-side.
